convention.md

```diff
@@ -364,7 +370,7 @@ the formats for displaying values.
 | color        | yes      |          | A comma-separated list of color formats supported; `rgb`, `hsv`, and/or `xyz`. The formats should be listed in order of preference (most preferred first, least preferred last). See the [color type](#color) for the resulting value formats. E.g. a device supporting RGB and HSV, where RGB is preferred, would have its format set to `"rgb,hsv"`. |
 | boolean      | no       | `false,true` | Identical to an enum with 2 entries. The first represents the `false` value and the second is the `true` value. Eg. `close,open` or `off,on`. If provided, then both entries must be specified. **Important**:  the format does NOT specify valid payloads, they are descriptions of the valid payloads `false` and `true`. |
 | json         | no       | `{"anyOf": [{"type": "array"},{"type": "object"}]}` | A [JSONSchema](http://json-schema.org/) definition, which is added as a string (escaped), NOT as a nested json-object. See [JSON considerations](#json-considerations), for some ideas wrt compatibility. If a client fails to parse/compile the JSONschema, then it should ignore the given schema and fall back to the default schema.
-
+| binary       | no       |  `application/octet-stream` | Any media type (also known as MIME type) that indicates the nature and the format of the data. [IANA Media Types](https://www.iana.org/assignments/media-types/media-types.xhtml) is the standardized and official data source for possible values. 
```

Comment:
I don't like this format attribute for the blob. What if the value accepts 2 types; eg. JPG and PNG?

What if those formats cannot be distinguished from the payload itself? Then we'd need to specify the format when setting the values in "the binary blob just set is image/jpg", and we cannot do that.

So I think we should drop the format here. A blob is a blob, without any further rules/policies. The consumer must use proper validations to ensure it is valid.